src/Components/Notifications/NotificationsWrapper.tsx

```diff
@@ -8,7 +8,7 @@ import { MarkAllAsReadPanelProps } from "Components/Notifications/MarkAllAsReadP
 const DROPDOWN_HEADER_HEIGHT = 113
 const VERTICAL_OFFSET = 10
 const DROPDOWN_CONTENT_HEIGHT =
-  DROPDOWN_HEADER_HEIGHT + DESKTOP_NAV_BAR_TOP_TIER_HEIGHT + VERTICAL_OFFSET
+  DROPDOWN_HEADER_HEIGHT + DESKTOP_NAV_BAR_HEIGHT + VERTICAL_OFFSET
```

Comment:
Before this change, the maximum height was too large, so it was possible to scroll out of the container.

Reply:
Makes sense, we made some changes here so it could've broke it.